### ISSUE TITLE
Unload, Flag 'n more

### ISSUE THREAD

**Issue by mrochaaa** [erosman/support#79]
3 suggestions:

Please add unload tabs in tabs context menu, like unload to right, unload to left etc. 

Add in the Flag tooltip, domain country, http2 status, and with click an redirector function like rediretor of Flagfox

Add counter style svg icon with bold font

thank you

**Comment by erosman** [comment]
> Please add unload tabs in tabs context menu, like unload to right, unload to left etc.

I have to look into it.. Firefox 67+ automatically unloads/hibernates tabs.

> Add in the Flag tooltip, domain country, http2 status, and with click an redirector function like rediretor of Flagfox

That is possible if there is a popular demand for a popup. Frankly, I never used those features on FlagFox (when I was a user).
ATM, the mouseover shows IP, country & continent

> Add counter style svg icon with options to bold font e font and background colors like Tab counter plus
That addon only has a counter feature. rebuilding the svg on every tab add/remove takes a lot of resource.

**Comment by mrochaaa** [comment]
I said add in mouseover not just ip country, but domain country, about redirector popup, I used it a lot on Flagfox legacy, now I use "Redirect Link", but if included on Foxy Tab I can use just one plugin and use the flag icon, I use a lot to Archive.org and Google Cache. Thank you.

**Comment by erosman** [comment]
>domain country

What do you mean?

Like if it is .uk to say United Kingdom?

> redirector popup

I am not sure what you mean. What should it do?

**Comment by mrochaaa** [comment]
for instance if the url is "www.example.uk" and ip is from USA show:

140.82.114.3 = United States
Domain = United Kingdom

about redirector, what I mean, is add an left click action to flag button to do the same fuction of extension "Redirect Link"

https://addons.mozilla.org/en-US/firefox/addon/redirect-link/

**Comment by erosman** [comment]
> about redirector, what I mean, is add an left click action to flag button to do the same fuction of extension "Redirect Link"

Redirect which link? The tab link? All the time?

**Comment by mrochaaa** [comment]
No, the behavior is exactly like Flagfox, the URL of corrent tab, redirect on demand with right click in the flag I choose an custom service, google cache, archive.org etc

**Comment by erosman** [comment]
I see... OK ... if there is a popular demand for it, I will add it

**Comment by mrochaaa** [comment]
this issue is an neat popular demand...

**Comment by erosman** [comment]
**Discard** Tab, All, Left, Right is added to next release v2.16

**Comment by erosman** [comment]
Redirect feature added to v2.16
